zio-http/src/main/scala/zio/http/internal/HeaderGetters.scala

```diff
@@ -37,6 +38,18 @@ trait HeaderGetters { self =>
       parsed.toOption
     }
 
+  final def headers(headerType: HeaderType): Seq[headerType.HeaderValue] =
```

Comment:
The rule is:

Accept the most generic collection type, provide the most specific collection type.

So here, maybe use Chunk so the user has some guarantee about the resulting Seq.